server/apps/publish/publish_filters/filter_condition.py

```diff
@@ -0,0 +1,275 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import logging
+import re
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+from superdesk.utils import ListCursor
+from superdesk import get_resource_service
+from superdesk.errors import SuperdeskApiError
+from superdesk.io.subjectcodes import get_subjectcodeitems
+
+logger = logging.getLogger(__name__)
+
+
+class FilterConditionResource(Resource):
+    schema = {
+        'name': {
+            'type': 'string',
+            'nullable': False,
+        },
+        'field': {
+            'type': 'string',
+            'nullable': False,
+            'allowed': ['anpa-category',
+                        'urgency',
+                        'keywords',
+                        'priority',
+                        'slugline',
+                        'type',
+                        'source',
+                        'headline',
+                        'body_html',
+                        'genre',
+                        'subject'],
+        },
+        'operator': {
+            'type': 'string',
+            'allowed': ['in',
+                        'nin',
+                        'like',
+                        'notlike',
+                        'startswith',
+                        'endswith'],
+            'nullable': False,
+        },
+        'value': {
+            'type': 'string',
+            'nullable': False,
+        }
+    }
+
+    additional_lookup = {
+        'url': 'regex("[\w,.:-]+")',
+        'field': 'name'
+    }
+
+    datasource = {'default_sort': [('_created', -1)]}
+    privileges = {'POST': 'publish_filters',
+                  'PATCH': 'publish_filters',
+                  'DELETE': 'publish_filters'}
+
+
+class FilterConditionService(BaseService):
+    def on_create(self, docs):
```

Comment:
Maybe we should check that the sent filters respect the field restrictions: https://github.com/superdesk/superdesk/pull/732/files#diff-5ea451f73e5859fe28b57fc6c62d5fceR221